pyroclast/adversarial/adversarial.py

```diff
@@ -0,0 +1,48 @@
+import importlib
+import tensorflow as tf
+import matplotlib.pyplot as plt
+
+from pyroclast.common.tf_util import load_model
+from pyroclast.common.plot import plot_images
+from pyroclast.common.adversarial import fast_gradient_method
+from pyroclast.features.features import build_savable_objects
+
+
+def visualize_perturbation(data_dict, seed, output_dir, debug, module_name,
+                           model_name, norm, data_index, epsilon, **kwargs):
+    module = importlib.import_module(module_name)
+    model = load_model(module,
+                       model_name,
+                       data_dict,
+                       output_dir=output_dir,
+                       **kwargs)
+    for batch_data in data_dict['train']:
+        x = batch_data['image'][0]
+        y = batch_data['label'][0]
+        break
+    x = tf.cast(x, dtype=tf.float32) / 255.
+
+    x = tf.reshape(x, [1] + x.shape)
+    y = tf.reshape(y, [1] + y.shape)
+
+    num_classes = 10
+    feature_idx = 0
+    class_idx = 0
```

Comment:
Should these be hard coded?

Reply:
Obviously not. All of this is going to change.